src/tools/metadata-to-ipfs.ts

```diff
@@ -0,0 +1,41 @@
+import fs from "fs";
+// @ts-ignore
+import pinataSDK from "@pinata/sdk";
+import { NFTMetadata } from "../rmrk1.0.0/classes/nft";
+
+const pinata = pinataSDK(process.env.PINATA_KEY, process.env.PINATA_SECRET);
+
+const defaultOptions = {
+  pinataOptions: {
+    cidVersion: 1,
```

Comment:
Is this needed of can we let Pinata decide by default and pick whatever is current?

Reply:
I read here that cid version 1 is new standard and should be preferred now so I decided to stick with it:

https://docs.ipfs.io/concepts/content-addressing/